docs/apply/interactive/process_input.md

```diff
@@ -5,8 +5,8 @@ title: interactive / process_input
 
 ### NAME
 
-    process_input  -  provide a hook for the MudOS driver to send a copy of
-    what the player is typing to the player object.
+    process_input  -  provide a hook for the driver to send a copy of what the
```

Comment:
All of the other docs include `()` after the function names.

Reply:
done! thanks! 🤗